WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListActivity.java

```diff
@@ -49,71 +51,46 @@ public void onCreate(Bundle savedInstanceState) {
 
         setContentView(R.layout.post_list_activity);
 
-        Toolbar toolbar = findViewById(R.id.toolbar);
-        setSupportActionBar(toolbar);
 
```

Comment:
Just a thought: It might be worth converting this activity to Kotlin in a separate small PR.

Reply:
I agree. At first I wanted to convert it in a small PR and merge it before this one. However, I felt it'd be waste to review a code which I'd have deleted in this PR. So I decided to take the opposite approach and I'm planning to create the PR when this one is merged.